manifests/init.pp

```diff
@@ -23,7 +23,7 @@
   $repo_proxy_password                                 = $nodejs::params::repo_proxy_password,
   $repo_proxy_username                                 = $nodejs::params::repo_proxy_username,
   Optional[String] $repo_release                       = $nodejs::params::repo_release,
-  $repo_url_suffix                                     = $nodejs::params::repo_url_suffix,
+  $repo_version                                        = $nodejs::params::repo_version,
```

Comment:
If we're doing breaking changes, perhaps change this to have a data type?
```suggestion
  String[1] $repo_version                              = $nodejs::params::repo_version,
```

Reply:
sure